### DISCUSSION TITLE
"Building ESAPI" link is dead

### DISCUSSION THREAD

**Discussion by noloader** [ESAPI/esapi-java-legacy#726] [General]
Webmaster strikes again: https://github.com/ESAPI/esapi-java-legacy/blob/d6251b5e5ade0bfe312efbb1d798af266ec70e0d/CONTRIBUTING-TO-ESAPI.txt#L57

**Comment by kwwall** [comment]
Note to self: Original page is at - https://wiki.owasp.org/index.php/ESAPI-Building

Just move to local GitHub wiki and update CONTRIBUTING-TO-ESAPI.txt to refer to the GH wiki link.

**Comment by kwwall** [comment]
@noloader - Now under https://github.com/ESAPI/esapi-java-legacy/wiki/Building-ESAPI. I updated the CONTRIBUTIING-TO-ESAPI.txt file as well. Thanks for noticing.
